pylint/checkers/imports.py

```diff
@@ -177,7 +177,7 @@ def _repr_tree_defs(data, indent_str=None):
 def _dependencies_graph(filename, dep_info):
```

Comment:
```suggestion
from typing import dict

def _dependencies_graph(filename: str, dep_info: Dict[...]) -> ... :
```
Regarding my comment about adding typing, it would be adding something like this on modified function.